app/helpers/application_helper.rb

```diff
@@ -1,5 +1,11 @@
 module ApplicationHelper
 
+  def beta_user?(user)
+    return true unless ENV['SHF_BETA'] == 'yes'
+    
+    BETA_USERS.include?(user.email)
```

Comment:
Some  thoughts about the Beta Testing 

- Good idea to set up the ENV for beta testing. If we need to do more with this, this is a great start

- Will an _admin_ always be able to do beta testing stuff, too?  
In looking through your code to add the beta testing, it seems you're always checking to see
 ``` 
     if a user is a beta tester or admin
          then they can do/see whatever....
```

- Whether or not someone can do something is an _authorization (permission)_ thing.  We're using Pundit for that; Pundit has the responsibility for checking who (users) can/cannot do things/access things.  So this (can someone access/do something for that should be restricted to beta testers?)  should also use Pundit. 

 You can call Pundit methods from views.  We can define the pundit policy method in ApplicationPolicy.  Within the ApplicationPolicy class we can have a method that will probably be like:

```
  def beta_tester?
    user.admin? || beta_user?(user)
  end
```

There is a little more code (maybe 1 more method in ApplicationPolicy).  I can help with this. 

This will also take the 'who is a beta tester?' responsibility out of the Helper class.  I don't think it really belongs there.  (Although it does fall into 'pulling logic out of Views' category.)

Reply:
My approach to the beta testing is this:

1) This is a very short (1-3 days) testing period
2) Adjustments to production code should be as light as possible and easily removable in a subsequent, _small_ PR
3) We should be able to stop the beta program very easily, and _without_ any code changes nor a new deployment
4) All authorization is already covered by Pundit, and I don't need that for this beta.  The small hooks I've put in place are solely for managing _visibility_ to page elements which are available only to beta testers.

So, I want to keep this the way that I have it, as this is consistent with the above objectives.

We can engineer a more robust(?) way to manage beta programs if this is needed repeatedly.  However, this PR is not the appropriate vehicle for that.

Reply:
"Will an admin always be able to do beta testing stuff, too?
In looking through your code to add the beta testing, it seems you're always checking to see"

Yes, we have defined the beta such that admins have visibility to beta elements in addition to regular beta users.  (I think it is moot for this beta, and Susanna is in the list of beta users (and may be the only admin?)).

Also, I added a check for admin? to just one place in the code for beta visibility.  The second place already checked for admin? as part of regular processing.

Reply:
"You can call Pundit methods from views. We can define the pundit policy method in ApplicationPolicy." 

Yes, we can call pundit from views, but in this case I need to call my help in one instance with the `user` argument == `current_user`, and in another with `user` == `@user` (where, in fact, `current_user` and `@user` may be two different people).  Pundit presents challenges to the latter approach, although I am sure I could engineer it to work if needed.

Reply:
There's an easy pundit method for this:
```
if policy(@company).beta_access?(current_user)....
```

But as I mentioned in today's team meeting, we can definitely make these changes in another PR.